cerebrium/conduit/introduction.mdx

```diff
@@ -73,6 +73,7 @@ In the Conduit object, there are the following parameters:
 - **MEMORY**: This is the number of GB of memory you'd like to allocate to
   your model. Optional as it defaults to 8.0GB. Depending on your hardware
   selection, this float can be between 2.0 and 256.0
+- **COOLDOWN**: Cooldown period in seconds before an inactive replica of your deployment is scaled down. Defaults to 60s.
```

Comment:
Maybe mention that this starts once a request/workload has finished

Reply:
At the moment it starts once the request is recieved.

Reply:
So that wouldn't be the case